default.nix

```diff
@@ -0,0 +1,26 @@
+{ pkgs ? (import <nixpkgs> {})
+, ocamlPackages ? pkgs.ocamlPackages
+}:
+
+with pkgs;
+
+stdenv.mkDerivation rec {
+
+  name = "coq_website";
+  src = null;
+
+  buildInputs = [
+    dune
+    lua51Packages.lua
```

Comment:
Is my beloved Lua still used?

Reply:
Yes, because the `archive2web` script is still there for now.